gradle-conjure/src/test/groovy/com/palantir/gradle/conjure/ConjurePluginTest.groovy

```diff
@@ -656,62 +656,6 @@ class ConjurePluginTest extends IntegrationSpec {
         'peer'     | ''
     }
 
-    def 'sets up idea source sets correctly'() {
-        given:
-        createFile('api/api-jersey/some-extra-source-folder')
-
-        file('build.gradle') << '''
-        subprojects {
-            apply plugin: 'idea'
-
-            idea {
-                module {
-                    sourceDirs += file('some-extra-source-folder')
-                }
-            }
-        }
-        '''.stripIndent()
-
-        when:
-        runTasksSuccessfully('idea')
-
-        then:
-        def slurper = new XmlParser()
-        def module = slurper.parse(file('api/api-jersey/api-jersey.iml'))
-        def sourcesFolderUrls = module.component.content.sourceFolder.@url
-
-        sourcesFolderUrls.size() == 2
-        sourcesFolderUrls.contains('file://$MODULE_DIR$/some-extra-source-folder')
-        sourcesFolderUrls.contains('file://$MODULE_DIR$/build/generated/sources/conjure-jersey/java/main')
-    }
-
-    def 'compileTypeScript is run on build for circle node 0'() {
-        when:
-        def stdout = runTasksSuccessfully('build', '--dry-run',
-                '-P__TESTING_CIRCLE_NODE_INDEX=0').standardOutput
-
-        then:
-        stdout.contains ':api:compileTypeScript SKIPPED'
-    }
```

Comment:
I'm pretty sure we want to keep this behaviour. More details at https://github.com/palantir/gradle-conjure/pull/1349, but when rolling out Gradle 8 we kept breaking publish, so we want to make sure that all tasks that get run at publish time happen as a dependency of `build`, to ensure they get tested out pre-merge. Otherwise you get excavators that merge but then break publish.

Reply:
Do we think the `isCircleNode0OrLocal` filtering is really necessary? It's sort of strange for task dependencies to be different on different CI nodes.

I can't imagine the compiling TypeScript is that time consuming. Do we have repos where this requires a meaningful amount of CI resources?